here-naksha-app-service/src/test/resources/unit_test_data/ViewHandler/setup/create_features_base.json

```diff
@@ -0,0 +1,255 @@
+{
+  "type": "FeatureCollection",
+  "features": [
+    {
+      "id": "FeatId-create-5002",
+      "type": "Feature",
+      "properties": {
+        "@ns:com:here:mom:meta": {
+          "tid": "urn:here::here:Topology-Transaction:1866780",
+          "owner": "topology_adapter_service",
+          "sourceId": "Task_1",
+          "operation": "CREATED",
+          "modelVersion": "2.121.0",
+          "lastUpdatedBy": "topology_adapter_service",
+          "updatedByUser": "topology_adapter_service"
+        },
+        "speedLimit": "20",
+        "testSpace" : "base"
+      },
+      "geometry": {
+        "type": "Point",
+        "coordinates": [
+          8.68872,
+          50.0561,
+          292.94377758
+        ]
+      }
+    },
+    {
+      "id": "FeatId-create-5003",
+      "type": "Feature",
+      "properties": {
+        "@ns:com:here:mom:meta": {
+          "tid": "urn:here::here:Topology-Transaction:1866780",
+          "owner": "topology_adapter_service",
+          "sourceId": "Task_1",
+          "operation": "CREATED",
+          "modelVersion": "2.121.0",
+          "lastUpdatedBy": "topology_adapter_service",
+          "updatedByUser": "topology_adapter_service"
+        },
+        "speedLimit": "20",
+        "testSpace" : "base"
+      },
+      "geometry": {
+        "type": "Point",
+        "coordinates": [
+          8.68872,
+          50.0561,
+          292.94377758
+        ]
+      }
+    },
+    {
+      "id": "FeatId-create-5003",
```

Comment:
Id "FeatId-create-5003" repeats in this feature list. Pls remove one of them or correct their IDs.